CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
+* test: Delete Istio resources if install does not complete (Backport PR #16829, Upstream PR #16440, @jrajahalme)
+* test: do not useDigest in upstream tests (Backport PR #16829, Upstream PR #16886, @aanm)
+* test: Move instrumentation to AfterFailed instead of AfterAll (Backport PR #16829, Upstream PR #16845, @christarazi)
+* test: Redeploy DNS after endpointRoutes reconfiguration (Backport PR #16829, Upstream PR #16767, @joestringer)
+* test: Wait for kube-dns before starting test (Backport PR #16829, Upstream PR #16411, @jrajahalme)
+* workflows: fix concurrency group names (Backport PR #16829, Upstream PR #16711, @nbusseneau)
+* workflows: fix L4LB test missing PR reporting on issue_comment (Backport PR #16829, Upstream PR #16830, @nbusseneau)
+* workflows: fix Relay pgrep check when using additional flags (Backport PR #16829, Upstream PR #16831, @nbusseneau)
+* workflows: remove label filters for testing workflows (Backport PR #16829, Upstream PR #16735, @nbusseneau)
+* workflows: various fixes & consistency passes (Backport PR #16829, Upstream PR #16787, @nbusseneau)
+
+**Misc Changes:**
+* .github: do not useDigest in conformance tests (Backport PR #16837, Upstream PR #16836, @aanm)
+* [v1.10] install/kubernetes: re-add restartPods (#16858, @aanm)
+* Allow configuration of probe timers in Helm chart (Backport PR #16774, Upstream PR #16584, @jonkerj)
+* bugtool: Collect BPF cgroup programs related information (Backport PR #16774, Upstream PR #16691, @aditighag)
+* build(deps): bump docker/setup-buildx-action from 1.4.1 to 1.5.0 (#16763, @dependabot[bot])
+* build(deps): bump docker/setup-buildx-action from 1.5.0 to 1.5.1 (#16856, @dependabot[bot])
+* conformance tests: Use hubble-relay-ci image (Backport PR #16829, Upstream PR #16363, @michi-covalent)
+* contrib/docs: rename 'cilium-actions.yml' with 'maintainers-little-helper.yaml (Backport PR #16774, Upstream PR #16750, @aanm)
+* contrib: Explicitly set remote for backport branches (Backport PR #16829, Upstream PR #16804, @twpayne)
+* docs(k3s): add back the flag to disable network policies (Backport PR #16829, Upstream PR #16755, @rio)
+* docs: account for bandwidth manager now being disabled by default (Backport PR #16829, Upstream PR #16782, @bmcustodio)
+* docs: Document dns visibility limitations (Backport PR #16829, Upstream PR #16822, @joestringer)
+* docs: fix code-block for bpf mount example (Backport PR #16774, Upstream PR #16719, @aanm)
+* github: Increase workflow timeout (Backport PR #16829, Upstream PR #16819, @jrajahalme)
+* Improve logging when cgroupfs mount fails (Backport PR #16829, Upstream PR #15999, @johngv2)
+* pkg/k8s: re-add CiliumIsUp Node condition even if removed (Backport PR #16829, Upstream PR #16857, @aanm)
+* Revert "docs: add 'endpointRoutes.enabled=true' to aws-cni" (Backport PR #16774, Upstream PR #16756, @bmcustodio)
+* Revert "policy: Make selectorcache callbacks lock-free" (Backport PR #16829, Upstream PR #16769, @aanm)
```

Comment:
Isn't this part of a bugfix in the end? Our CI failed for a legitimate reason, no?

Reply:
Yes-and-no the bug fix will be this one: `* Potential deadlock in pod identity updates has been fixed. (Backport PR #16829, Upstream PR #16801, @jrajahalme)`. Although we have revert it, we have fixed the bug again.